Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -95,6 +96,8 @@ public void Shutdown()
         {
             if (initialized)
             {
+                Stopped.Invoke();
```

Comment:
We probably want to disconnect the clients first no?

Reply:
fixed